.idea/misc.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove all .idea files, as they are your own Intellij configs and not the project you should add .idea into the gitignore file